upup/models/cloudup/resources/addons/networking.weave/k8s-1.12.yaml.template

```diff
@@ -36,6 +36,14 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
```

Comment:
Does weave actually fetch resources using the deprecated api groups? If not, this is not necessary. It doesn't do any harm either though.

Reply:
Best to keep it as close as possible to their official manifest, I guess. Thanks for checking this.
https://github.com/weaveworks/weave/blob/9f00f78d3b9d5a8a31fdd90ec691095028e2690a/prog/weave-kube/weave-daemonset-k8s-1.11.yaml#L28-L35